### PR TITLE
chore: change placeholder text for foreground input

### LINKED ISSUE #107: Change placeholder text for the foreground color input
This is what we have right now for the input placeholders. 
Screen Shot 2022-09-02 at 11 47 16 AM

But that implies to the user that you can input mixed color formats and receive a contrast ratio.
Right now our application does not support that functionality, so will need to change the messaging.

Change the foreground input placeholder so it shows an example hex code value instead of a rgb value.
Screen Shot 2022-09-02 at 11 52 52 AM

### PR DESCRIPTION
**Pull request by thetalesofj**
Foreground input placeholder was changed from an rgb value to hex code.

# Description

Fixes #107 - placeholder text changed from rgb value to hex code

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore items (this includes basic clean up of files, or updates to packages)
- [ ] Updates to documentation

## Checklist

- [x] I have a descriptive title for my PR
- [x] I have linked this PR to the corresponding issue. [See how to do that here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] I have run the project locally and reviewed the code changes
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jdwilkin4** [comment]
@thetalesofj 

I have updated the title of your PR.
When naming PR is it important to use descriptive names.
It will make it easier for the team to understand what you are working on at quick glance as well as when the team looks through the git history of what was worked on in the past.

**Comment by thetalesofj** [comment]
Hi Jessica, I appreciate the advice and I'll make sure to keep it to the format you used

**Review by AndreyDmitriev39r — CHANGES_REQUESTED** [review]
html is cool,
why pr template is changed - I don't know.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Fixes # (issue)
 
 ## Checklist
 
-- [ ] I have a descriptive title for my PR
-- [ ] I have linked this PR to the corresponding issue. [See how to do that here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
-- [ ] I have run the project locally and reviewed the code changes
-- [ ] My changes generate no new warnings
```
Comment (AndreyDmitriev39r) [inline .github/pull_request_template.md:24]: Why is this file changed. It has nothing to do with #107 issue.
Please clarify or undo changes for this file.

**Review by AndreyDmitriev39r — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (thetalesofj) [reply .github/pull_request_template.md:24]: I was having trouble trying to undo the ticked boxes on the last pr and made the change. If there is a way to delete this then that would be great

Comment (AndreyDmitriev39r) [reply .github/pull_request_template.md:24]: you can simply edit `pull_request_template.md` file

just make it like it was before your last commit.

and then push such changes to this PR.

Comment (thetalesofj) [reply .github/pull_request_template.md:24]: I already pushed this change yesterday and it should be in the commit section 'update pull_request_template.md

Comment (AndreyDmitriev39r) [reply .github/pull_request_template.md:24]: I get it. but the thing is
If you will take look at the `files changed` tab right now - you will see that you did not undo changes exactly right. 4 lines - 21, 22, 23, 24 - are still considered changed. Originally there was a space between the brackets. In your last version of code there is no space between the brackets.
So add such spaces and push it to this PR.

**Review by sparkO7 — APPROVED** [review]
LGTM 👍

**Review by AndreyDmitriev39r — APPROVED** [review]
LGTM!

**Comment by thetalesofj** [comment]
Appreciate it guys!
